File: C#/begginer/1020.cs
using System; 

class URI {

    static void Main(string[] args) { 

        int n = int.Parse(Console.ReadLine());

        int year = n / 365;
        n -= year * 365;
        int month = n / 30;
        n -= month * 30;
        int day = n;

        Console.WriteLine($"{year} ano(s)");
        Console.WriteLine($"{month} mes(es)");
        Console.WriteLine($"{day} dia(s)");

    }

}

File: Java/begginer/1020.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);
        
        int input = scanner.nextInt();
        int year = input / 365;
        input = input % 365;
        int month = input / 30;
        input = input % 30;
        
        System.out.println(year + " ano(s)");
        System.out.println(month + " mes(es)");
        System.out.println(input + " dia(s)");
    }
 
}
